routes/baseDBMethods.js

```diff
@@ -22,6 +24,35 @@ function handleDocResponse(ctx, doc) {
   }
 }
 
+function proccessDatesRangeParameters(ctx) {
+  const startDate = parseInt(ctx.query.startDate, 10) || null;
+  const endDate = parseInt(ctx.query.endDate, 10) || null;
+
+  const datesFilter = {};
+
+  if ((startDate != null) && (endDate != null) && (startDate > endDate)) {
+    throw new RangeError('Query parameter \'startDate\' can\'t be bigger than Query parameter \'endDate\'');
+  } else if ((startDate != null) || (endDate != null)) {
+    datesFilter.date = {};
+
+    if (startDate != null) {
+      if (startDate < 1) {
+        throw new RangeError('Query parameter \'startDate\' must be a positive number');
+      }
+      datesFilter.date.$gte = moment(startDate).tz('Israel').format();
```

Comment:
The proper timezone identifier for Israel is "Asia/Jerusalem" ([see here](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones) - referenced from moment-timezone documentation).